### PR TITLE
Feat/fix sidebar

### PR DESCRIPTION
**Pull request by CloudyMushroom**
# Description

Merge ko na hehe

Fixes # (issue)

# Type of change

Please delete options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

# Screenshots (if appropriate)

Add screenshots that could help the reviewer.

# Checklist:

-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings
-   [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by CloudyMushroom** [comment]
@yell66w pwede ko na merj to sir??

File: frontend/src/styles/globals.css [hunk 1]
```diff
@@ -58,6 +58,42 @@ code {
...
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 10%;
+  right: 0;
+  border: #4c00aa 0.3em solid;
+  background-color: #4c00aa;
+  border-top-right-radius: 1em;
+  transform: translate(1px, 36px);
+}
```
Comment (yell66w) [inline frontend/src/styles/globals.css:96]: gagi pre bat hotdog HAHAHA

**Review by yell66w — CHANGES_REQUESTED** [review]

File: frontend/src/styles/globals.css (hunk 1 above)
Comment (yell66w) [reply frontend/src/styles/globals.css:96]: ibahin mo name yung formal tignan

Comment (CloudyMushroom) [reply frontend/src/styles/globals.css:96]: ay gagi oo nga pala HAHAHAHAHHA

**Comment by CloudyMushroom** [comment]
@yell66w Removedd na

**Pull request merged**